cli/src/opts/cast.rs

```diff
@@ -400,10 +401,13 @@ Examples:
     #[clap(visible_alias = "e")]
     #[clap(about = "Estimate the gas cost of a transaction.")]
     Estimate {
-        #[clap(help = "The destination of the transaction.", parse(try_from_str = parse_name_or_address), value_name = "TO")]
-        to: NameOrAddress,
+        // #[clap(help = "The destination of the transaction.", parse(try_from_str = parse_name_or_address), value_name = "TO")]
+        #[clap(skip)]
+        to: Option<NameOrAddress>,
```

Comment:
@mattsse @OmkarB asked me how he can make Clap skip this positional argument if the `--create` flag is present, but I couldn't find anything myself. Is it possible?